Design/DesignDoc.md

```diff
@@ -0,0 +1,220 @@
+# Design Document
+
+## _AmazonBudgetService_ Design
+
+## 1. Problem Statement
+
+Amazon Elite Budget Service is a highly advanced budgeting app that will enable users to have more control over their money/expenditures.
```

Comment:
I would probably not include "Amazon" in the name of the project. It's using AWS which is an Amazon service, but if there's nothing specific about Amazon in terms of its functionality I wouldn't include that.

I'd also consider how you position it - seeing "a **highly advanced** budgeting app" sets some pretty high expectations.